assets/blocks/course-outline/module-block/module-edit.js

```diff
@@ -158,6 +164,7 @@ export const ModuleEdit = ( props ) => {
 							onChange={ updateName }
 						/>
 					</h2>
+					{ teacher && <span>({ teacher })</span> }
```

Comment:
I think the style of the new teacher section could be improved a bit. Right now it looks odd when the module section is too long and the font blends in with the module's font.

<img width="649" alt="Screenshot on 2022-05-14 at 01-42-29" src="https://user-images.githubusercontent.com/1612178/168398452-2c5c7da3-381e-49ba-9f39-04642c174068.png">


What do you think about adding a bit of padding to the left and maybe changing the font size (make it smaller)? I'm no UI expert, so maybe there is a better approach here. :-)

Reply:
- Thanks @m1r0 ! Added a small padding to the left and made a text a little smaller [here](https://github.com/Automattic/sensei/pull/5114/commits/c285803478f6739ef2d0e0d33badb9bcc2c3bc20)